Core/Core/Inbox/ComposeMessage/View/ComposeMessageView.swift

```diff
@@ -307,6 +307,7 @@ public struct ComposeMessageView: View, ScreenViewTrackable {
             .font(.regular16, lineHeight: .condensed)
             .foregroundColor(.textDark)
             .padding(.vertical, 12)
+            .frame(maxHeight: .infinity, alignment: .center)
```

Comment:
I've asked Marci about this change, will update with his answer.
Regardless, please highlight these kind of unrelated changes in the PR description, especially if those are user facing.

Reply:
yes i faced an issues when you set much recipients in the list the 'TO' button in the top and '+' in the middle, so i fixed it 
@rh12 
<img src= "https://github.com/user-attachments/assets/446b112e-c31b-4c66-b4e4-dbe867af4b17">

Reply:
Marci confirmed it, that the "To" text should stay top aligned. Actually the "+" should also stay top aligned.
(This was probably not spotted in the previous PR)

Please revert the "To" change, and could you please fix the "+"? If so, please add that to the PR description.

Reply:
Thanks will do